### DISCUSSION TITLE
Disabling Response Caching in Langflow Chat UI: Exploring Options

### DISCUSSION THREAD

**Discussion by sri-dhurkesh** [langflow-ai/langflow#841] [Q&A]
Hi, I've been using Langflow for about a month now. I've noticed that in a chat UI, when I ask the same question that I've already asked before, it provides the same answer as before. Is there a way to disable the caching of responses in the Langflow chat UI?

**Comment by carlosrcoelho** [comment]
Hey!

Thanks for sharing your question here. I'm sorry for not getting back to you sooner.
Langflow has gone through some major changes since then. I recommend trying out the new version for the best experience. If you have any issues, please open a new one; we'll be here to help.
